src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -1827,6 +1827,11 @@ private CSDiagnosticInfo NotFound(CSharpSyntaxNode where, string simpleName, int
                 diagnostics.Add(ErrorCode.ERR_CycleInTypeForwarder, location, fullName, forwardedToAssembly.Name);
             }
 
+            return SingleTypeNameNotFound(whereText, diagnostics, location, forwardedToAssembly);
```

Comment:
What about other errors reported in this function: ERR_DottedTypeNameNotFoundInNS, ERR_DottedTypeNameNotFoundInNSFwd, etc. ?